ns1/__init__.py

```diff
@@ -478,14 +478,15 @@ def createReservation(
         dhcp_options=None,
         callback=None,
         errback=None,
+        **kwargs
```

Comment:
Can you add the docstring comment that other methods in the class have letting users know where list of valid keywords to pass is?